bundles/core/src/main/java/com/adobe/cq/commerce/core/cacheinvalidation/spi/package-info.java

```diff
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-@Version("1.0.0")
+@Version("0.0.1")
```

Comment:
Can you add javadoc for what is the intent of this package also mentioning that it's experimental.